.circleci/config.yml

```diff
@@ -33,24 +33,24 @@ workflows:
           name: 👮 lint
           <<: *executor
           context: code-validator
-      
+
       - python/run:
           name: 👮 mypy
           run: mypy .
           <<: *executor
           context: code-validator
-      
+
       - test:
           name: 🧪 test py<<matrix.python_version>> sqlalchemy-<<matrix.sqlalchemy_version >> pydantic-<<matrix.pydantic_version>> <<matrix.asyncpg>> <<matrix.aws_rds_iam>>
           context: code-validator
           matrix:
             parameters:
-              python_version: ["3.9", "3.10", "3.11", "3.12"]
+              python_version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
Should we keep testing with python 3.9 if we still support it?

Reply:
The test matrix would be too big afaik

Reply:
So should we keep supporting 3.9? I would say no.

Reply:
> The test matrix would be too big afaik

Too big for CCI to load you mean?

> So should we keep supporting 3.9? I would say no.

Python 3.9 EOL is on 31 Oct 2025, so there's technically almost a year of support for it. I guess it's up to us to decide if we want support still valid python versions or not. On Dialogue side, looks like we only one or two repo that still use 3.9, but we support it in all our libs.

Reply:
I put it back and cci is actually :ok: with it